src/Game/UI/Gumps/Login/LoginGump.cs

```diff
@@ -226,14 +226,21 @@ public LoginGump(LoginScene scene) : base(0, 0)
             offsetX += 7;
 
             // Text Inputs
-            Add(_textboxAccount = new StbTextBox(5, 16, 190, false, hue: 0x034F)
+            var savedAccounts = AccountManager.GetAccountNames(Settings.GlobalSettings.IP);
+            Add(_textboxAccount = new StbTextBoxCombo(savedAccounts, 5, 16, 190, false, hue: 0x034F, maxComboHeight : 100)
             {
                 X = offsetX,
                 Y = offsetY,
                 Width = 190,
                 Height = 25,
             });
 
+            _textboxAccount.OnOptionSelected += (sender, e) =>
```

Comment:
I was looking for a place to unhook this but I couldn't find a good spot.  It would probably be a good idea for memory management.

Reply:
I removed the anonymous function and make it named function.  Overrode disposed, and unhooked it there.